CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Bend 
+Thank you for considering contributing to Bend!
+
+## How to Contribute
+### Reporting bugs
+1. **Check for existing issues:** Before you create a new issue, please do a search in [our issues](https://github.com/HigherOrderCO/Bend/issues) to see if the issue or feature request has already been filed.
```

Comment:
Should we also mention the HVM issues page? Since some issues might be there but not in the Bend repo

Reply:
true, that could be good, since we also have the "HVM issue" button